config/config.example.yml

```diff
@@ -3,21 +3,16 @@
 # This is the config.yml to use when using the
 # Datasets::Hathitrust::Configuration.
```

Comment:
I would be in favor of removing this file altogether as we already have the integration test config.  The example config for other projects has been functional for the external components in an out of the box fashion where you could copy config.yml.example to config.yml and it works in the demo case in the readme.